tests/Agent/IntegrationTests/ContainerApplications/AwsSdkTestApp/AwsSdkExerciser/AwsSdkExerciser.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+
+namespace AwsSdkTestApp.AwsSdkExerciser
+{
+    public class AwsSdkExerciser : IDisposable
+    {
+        public AwsSdkExerciser(AwsSdkTestType testType)
+        {
+            switch (testType)
+            {
+                case AwsSdkTestType.SQS:
+                    _amazonSqsClient = GetSqsClient();
+                    break;
+                default:
+                    throw new ArgumentException("Invalid test type");
+            }
+        }
+        #region SQS
+
+        private readonly AmazonSQSClient _amazonSqsClient;
+        private string _sqsQueueUrl = null;
+
+        private AmazonSQSClient GetSqsClient()
+        {
+            // configure the client to use LocalStack
+            var awsCredentials = new Amazon.Runtime.BasicAWSCredentials("dummy", "dummy");
+            var config = new AmazonSQSConfig
+            {
+                ServiceURL = "http://localstack-main:4566",
```

Comment:
Is this always the port for SQS or is it configured somewhere?

Reply:
It matches the configuration from `docker-compose-awssdk.yml` at https://github.com/newrelic/newrelic-dotnet-agent/pull/2577/files#diff-829b001fbb5f9dd93a0521fdfc59e62fabedf537c7fd59e530e43a805a930944R28 - I suppose it could be parameterized, but it's not really necessary.